src/pudl/metadata/fields.py

```diff
@@ -4202,6 +4279,10 @@
         "constraints": {"enum": RELIABILITY_STANDARDS},
         # TODO: Might want to make this column more specific to outages: ex: outage calculation standard.
     },
+    "standard_industrial_classification": {
+        "type": "string",
+        "description": "The company's type of business.",
+    },
```

Comment:
Whose classification system is this coming from? NAICS?  Is it a freeform text field, or a well defined set of codes or categories?

Reply:
Looks like this is a [standalone system](https://en.wikipedia.org/wiki/Standard_Industrial_Classification) that was replaced by the NAICS, but is still in use by the SEC.

Reply:
Some more detail in the description would help make it clear that this is a real thing in the world that can be referenced.  E.g.

"Four-digit Standard Industrial Classification (SIC) code identifying the company's primary industry. SIC codes have been replaced by NAICS codes in many applications, but are still used by the SEC."

Is this a string because it can have leading zeroes like FIPS codes?

Under our naming conventions I think this would probably be an `id` e.g. `primary_purpose_id_naics` though `id_sic` is a bit inscrutable.  `industry_id_sic` ???

Can we constrain this like we do for the FIPS IDs?
```py
"constraints": {
    "pattern": r"^\d{4}$",
}
```